### DISCUSSION TITLE
Is the default action of WUA to upgrade packages after the initial WUA installation?

### DISCUSSION THREAD

**Discussion by EcoELS** [Weatherlights/Winget-AutoUpdate-Intune#20] [Q&A]
Hi

Sorry for this basic question, but I'm a bit short on testing devices. What I seem to have observed after deploying WinGet-AutoUpdate-Configurator through Intune (Microsoft Store app (new)) - before deploying a configuration profile with the imported admx file - is that WUA just started updating apps.

My fear now is that if I apply users or devices to the group that is both requiring the installation of WinGet-AutoUpdate-Configurator and applying the configuration profile and the device first installs the app before applying the configuration profile, WUA will update apps on the system that I absolutely do not want updated (more to the point, I have a whitelist defined in the configuratin profile).

Did I observer that correctly and is this intended?

**Comment by Weatherlights** [comment]
If no configuration is available WAUC will install with the default enterprise configuration which does not include install a scheduled task for automatic updates: -silent -DoNotUpdate -DisableWAUAutoUpdate -NoClean.

So WAUC will wait until your configuration is in place before it updates anything :).
